.pre-commit-config.yaml

```diff
@@ -36,7 +36,7 @@ repos:
 #     exclude: ^tests/.*$
 
 - repo: https://github.com/SINTEF/ci-cd
-  rev: 94e7f9936a3278aed1a656cb7e151dd89da95409  # frozen: v1
+  rev: d525ea0f069b6615aa352a7f385c21b31f29b985  # frozen: v1.3.4
```

Comment:
This has changed because there has been some issues with the CI-CD in other repositories and Jesper has changed something in the original. This still did not work though so I d not want to accept this change now.

Reply:
Casper has released CI-CD v1.3.5

Maybe we could try to update to that version?

Reply:
Dette vil også komme automatisk i næste opdateringsrunde. Haster ikke med dette vil jeg mene.